sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -153,9 +153,15 @@ case class RowDataSourceScanExec(
       pushedDownOperators.limit.map(value => "PushedLimit" -> s"LIMIT $value")
     }
 
-    Map(
-      "ReadSchema" -> requiredSchema.catalogString,
-      "PushedFilters" -> seqToString(markedFilters.toSeq)) ++
+    val filterOrPredicateInfo = if (pushedDownOperators.createdInV2) {
+      val pushedPredicates = seqToString(pushedDownOperators.pushedPredicates.map(_.describe()))
+      Some("PushedPredicates" -> pushedPredicates)
```

Comment:
It looks unnecessary to have a different key name for v2 predicates. I'm OK to keep using `PushedFilters`

Reply:
OK